en/airframes/airframe_reference.md

```diff
@@ -561,6 +561,11 @@ This page lists all supported airframes and types including
  <td style="vertical-align: top;">HIL Quadcopter X</td>
  <td style="vertical-align: top;"><p>Maintainer: Lorenz Meier <lorenz@px4.io></p><p><code>SYS_AUTOSTART</code> = 1001</p></td>
 
+</tr>
```

Comment:
Please remove from file - airframe stuff gets generated from source and automatically updated when your PR is accepted.